### PR TITLE
[Feature-Poc] OSD visualizations panel integration with dashboards container

### LINKED ISSUE #184: [FEATURE - PoC] Integrate OSD visualizations to Operational Panels
**Is your feature request related to a problem?**
Here's the summary: https://github.com/opensearch-project/dashboards-observability/issues/106
This issue tracks the dashboards component for initial POC

### PR DESCRIPTION
**Pull request by ps48**
### Description
OSD visualizations panel integration, using RGL with dashboards container and visualization embeddable. 

### Issues Resolved
#184 #106 

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/custom_panels/panel_modules/visualization_container/visualization_container.scss [hunk 1]
```diff
@@ -58,3 +58,28 @@
 .panels-title-text {
   @include euiTextTruncate;
 }
+
+.osd-viz-panel {
+  .euiPanel {
+    box-shadow: none !important;
```
Comment (mengweieric) [inline public/components/custom_panels/panel_modules/visualization_container/visualization_container.scss:64]: is there a better way to not use !important?

Comment (ps48) [reply public/components/custom_panels/panel_modules/visualization_container/visualization_container.scss:64]: Need to check on this. Usually, we create a new class for selectivity. That was not possible in this case as my class was on same level with EUI.

**Comment by anirudha** [comment]
lets merge this to a feature branch and close this PR ?

**Comment by ps48** [comment]
Not going ahead with the feature for now. Closing the draft PR

**Pull request closed without merging**